cloudify_marketplace/blueprint/scripts/configure_openstack.py

```diff
@@ -7,6 +8,7 @@
 from cloudify import ctx
 from cloudify_rest_client import CloudifyClient
 from cloudify.state import ctx_parameters as inputs
+from cloudify.exceptions import NonRecoverableError
```

Comment:
This should be here, but it is also imported on line 7, just above the fold. It shouldn't be there as well.